### DISCUSSION TITLE
Radio Style

### DISCUSSION THREAD

**Discussion by NazarAli** [filamentphp/filament#9363] [General]
i think must change the inline radio style
from this style
![mmm](https://github.com/filamentphp/filament/assets/7056487/64c049d6-fc9c-4119-9b72-ee0db905ff83)

to this style
![mm](https://github.com/filamentphp/filament/assets/7056487/13107a66-11c9-48c9-9268-13451a3f4710)

**Comment by NazarAli** [comment]
how can modify the style ? or the field ?

**Comment by VincvanAs** [comment]
Add a label and remove the->inline() part.

https://filamentphp.com/docs/3.x/forms/fields/radio#positioning-the-options-inline-with-the-label

**Reply by NazarAli** [reply]
when remove inline it appear vertical

**Reply by VincvanAs** [reply]
sorry should be ->inline(false)

**Reply by NazarAli** [reply]
->inline(false) this like remove ->inline
it same result vertical
